media-sound/denemo/metadata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>waebbl@gmail.com</email>
+	<name>Bernd Waibel</name>
+</maintainer>
+<!-- TODO: Talk to sound project or remove it
+<maintainer type="project">
+	<email>sound@gentoo.org</email>
+	<name>Gentoo Sound project</name>
+</maintainer>
+-->
+<maintainer type="project">
+	<email>proxy-maint@gentoo.org</email>
+	<name>Proxy Maintainers</name>
+</maintainer>
+<longdescription>
+	Denemo is a music notation editor which uses
+	<pkg>media-sound/lilypond</pkg> to typeset music notation with LaTeX.
+</longdescription>
+<use>
+	<flag name="alsa">
+		Enable alsa support through <pkg>media-libs/alsa-lib
```

Comment:
Please terminate `<pkg>` on the same line, i.e. don't make it multi-line.

Reply:
This has slipped through. I changed it.